modules/tensor_mechanics/include/materials/czm/SalehaniIrani3DCTraction.h

```diff
@@ -0,0 +1,37 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "CZMMaterialBase.h"
+class SalehaniIrani3DCTraction;
+template <>
+InputParameters validParams<SalehaniIrani3DCTraction>();
+/**
+ * Implementation of a simple non-stateful exponential traction separation law
```

Comment:
Implementation of the non-stateful exponential traction separation law proposed by